src/main/java/com/opams/arch/entities/ProductEntity.java

```diff
@@ -1,8 +1,38 @@
 package com.opams.arch.entities;
 
-import javax.persistence.Entity;
+import lombok.*;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Positive;
+import java.math.BigDecimal;
 
 @Entity
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
 public class ProductEntity {
+
+    @GeneratedValue
+    @Id
+    private long productId;
+
+    @NotNull
+    @Positive
+    private BigDecimal retailPrice;
+
+    private String size;
+
+    private String color;
+
+    private String place;
+
+    private String image;   //base64 string
+
```

Comment:
- I don't think that product have exactly one image.
- I would say it's not a good choice of persisting base64 encoded image in db row, even if it's, then it's better to you some binary type like octet. 
- What about referencing images as url(where they are persisted)